content/blog/2024-12-21-how-do-i-...-in-dt-5.0/index.md

```diff
@@ -0,0 +1,508 @@
+---
+title: "How do I ... in darktable 5.0?"
+author: "Ralf Brown"
+slug: howto-in-5.0
+date: 2024-12-20
+lede: falls.jpg
+lede_author: "Ralf Brown"
+tags:
+  - howto
+---
+
+Users new to darktable who have experience with other raw developers
+may have trouble figuring out how to do various tasks as a result of
+different names or workflows in darktable compared to their previous
+software.  The following entries are a quick reference to help you
+accomplish what you want to do.
+
+Have a tip not covered here?  Share it in the comments!
+
+### How do I...
```

Comment:
We should start at heading level 2, not 3.

Reply:
Will do once I have solutions to the other problems with getting things to display correctly.